src/v2/view-builder/views/SuccessView.js

```diff
@@ -5,7 +5,16 @@ import { loc } from 'okta';
 
 const Body = BaseForm.extend({
   title () {
-    return  loc('oie.success.redirect', 'login');
+    // For more info on the API response available in appState, see IdxResponse:
+    // https://github.com/okta/okta-core/blob/master/server/api/src/main/java/com/okta/api/mediation/
```

Comment:
we can just keep the file name `IdxResponseBuilder.java` instead of whole link..